index.html

```diff
@@ -4,10 +4,310 @@
-    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+
     <title>Portfolio desktop</title>
 </head>
 <body>
-    <h1>Hello world</h1>
+    
+    <div class="main">
+        <header>
+            <a href="##" alt="logo"> <span class="logo">RIDA</span></a>
+            <nav>
+                    <ul class="navigation-items">
+                            <li><a href="#">Portfolio</a></li>
+                            <li><a href="#">About</a></li>
+                            <li><a href="#">Contact</a></li>
+                        </ul>
+                <span class="sp2"><img src="./image/Union.png" alt=""></span></nav>
+        </header>
+            <section class="paragraph">
+               <h1 class="text">I'm RIDA<br>Glad to see you!</h1>  
+                <p class="text2">
+                   I’m a software developer!
+                    I can help you build a product
+                     feature or website Look through
+                      some of my work and experience!
+                       If you like what you see and 
+                       have a project you need coded,
+                        don’t hestiate to contact me.
+                </p>
+                <h2 class="a">LET’S CONNECT</h2>
+
+                   <ul class="social-media">
+                       <li> <a href="##"><img src="./image/twit.png" alt=""></a></li>
+                       <li><a href="##"><img src="./image/linked.png" alt=""></a></li> 
+                       <li><a href="##"><img src="./image/mail.png" alt=""></a></li> 
+                       <li><a href="##"><img src="./image/pp.png" alt=""></a></li> 
+                       <li><a href="##"><img src="./image/peace.png" alt=""></a></li>
+                   </ul>                 
+               </section>
+           
+               </div>
+          <div class="cards">
+        <section class="card001">
+            <div class="card1">
+          <div class="c001">
+            <img class="img2" src="./image/image2.png" alt="hello">
+          </div>
+          <div class="c002">
+        <h2>Tonic</h2> 
+          <ul>
+           <li>CANOPY</li>
+         <li><img src="./image/image3.png" alt="count1"></li>
+            <li>Back End Dev</li>
+            <li><img src="./image/image4.png" alt="count2"></li>
+            <li>2015</li>
+        </ul>
+        <p>
+            A daily selecion of privately personalized reads; no accounts or sign up required.
+        </p>
+         <ul class="lang">
+            <li><h5>html</h5></li>
+            <li><h5>css</h5></li>
+            <li><h5>javascript</h5></li>
+         </ul>
+         <button class="button" type="button">See Project</button>
+        </div>
```

Comment:
- [ ] To replicate the given template design as much as possible and to an overall better UI 👉🏽 Kindly try to match the given template when it comes to positioning and the space between the elements 👇🏽.

### Your design
![image](https://user-images.githubusercontent.com/43172164/208075790-1d217c78-9807-4325-9bdc-fa144671482c.png)

### Figma template
![image](https://user-images.githubusercontent.com/43172164/208074860-c2d82f43-e5e2-4036-a739-bdf825815907.png)

Reply:
- I didn't see anything to comment on but just improve with the structure always try to format your code for it to look clean